Initial Release Infrastructure Cmdlets

troettinger opened the pull request:
This is the initial release of the Infrastructure API Rest Wrapper to provide a cmdlet experience. Cmdlets will call FRP, HRP and URP.

azurecla commented:
Hi __@troettinger__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (thoroet@ntdev.microsoft.com). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

igorse commented:
- Verbs should start form capital letters
- Functions should be exported via export-member
- Recommend that customer uses Add-AzureStackAzureRmEnvironment from AzureStack.Connect module. Then your command-lets can just take the env name and assume it is created properly. You don't need to create env in every command-let.
- Remove unnecessary white space and align indentation
- Add some minimal synopsis to each function to describe its behavior
- Add #requires for AzureRm.AzureStackAdmin module

troettinger commented:
I addressed all your comments beside from one building a dependency on AzureStack.Connect.  I want to visit this at a later point. As scripts are shared outside Github as well once downloaded so I want to make this works
